packages/apputils-extension/style/base.css

```diff
@@ -6,3 +6,33 @@
 
 @import './notification.css';
 @import './splash.css';
+
+.jp-ContextualShortcut-TableItem {
```

Comment:
Could you please move the style in a specific file?

Reply:
Ok. This is done a specific file `packages/apputils-extension/style/contextualhelp.css`has been added and the style details moved to it.